en/enable-tidb-cluster-auto-scaling.md

```diff
@@ -0,0 +1,193 @@
+  tikv:
+    minReplicas: 3
+    maxReplicas: 4
+    metrics:
+      - type: "Resource"
+        resource:
+          name: "cpu"
+          target:
+            type: "Utilization"
+            averageUtilization: 80
+```
+
+The TiDB component can be configured using `spec.tidb`. Currently, the auto-scaling API of TiDB is the same as that of TiKV.
+
+In a `TidbClusterAutoScaler` object, the `cluster` attribute specifies the TiDB clusters to be auto-scaled. These clusters are marked by `name` and `namespace`. You need to provide the metrics collection and query service to `TidbClusterAutoScaler` because it grabs resource usage through the metrics collection component. The `monitor` attribute refers to the `TidbMonitor` object. For more information, see [Monitor TiDB Clusters using TidbMonitor](monitor-using-tidbmonitor.md).
+
+If you use the external `Prometheus` instead of `TidbMonitor`, you can fill in the Host by configuring `spec.metricsUrl` to specify the monitoring metrics collection service for the TiDB cluster. If you deploy the monitoring of the TiDB cluster using `Helm`, take the following steps to specify `spec.metricsUrl`.
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: TidbClusterAutoScaler
+metadata:
+  name: auto-scaling-demo
+spec:
+  cluster:
+    name: auto-scaling-demo
+    namespace: default
+  metricsUrl: "http://<release-name>-prometheus.<release-namespace>.svc:9090"
+  ......
+```
+
+## Quick start
+
+Run the following commands to quickly deploy a TiDB cluster with 3 PD instances, 3 TiKV instances, 2 TiDB instances, and the monitoring and the auto-scaling features.
+
+```shell
+$ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/auto-scale/tidb-cluster.yaml -n <namespace>
+tidbcluster.pingcap.com/auto-scaling-demo created
+
+$ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/auto-scale/tidb-monitor.yaml -n <namespace>
+tidbmonitor.pingcap.com/auto-scaling-demo created
+
+$ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/auto-scale/tidb-cluster-auto-scaler.yaml  -n <namespace>
+tidbclusterautoscaler.pingcap.com/auto-scaling-demo created
+```
+
+When you create the TiDB cluster, you can stress test the auto-scaling feature through database stress test tools such as [sysbench](https://www.percona.com/blog/tag/sysbench/).
+
+Run the following commands to destroy the environment:
+
+```shell
+kubectl delete tidbcluster auto-scaling-demo -n <namespace>
+kubectl delete tidbmonitor auto-scaling-demo -n <namespace>
+kubectl delete tidbclusterautoscaler auto-scaling-demo -n <namespace>
+```
+
+## TidbClusterAutoScaler configurations
+
+1. Set the auto-scaling interval.
+
+    Compared with the stateless web service, a distributed database software is often sensitive to the instance auto-scaling. You need to make sure that there is a certain interval between each auto-scaling in case it is too often.
```

Comment:
```suggestion
    Compared with the stateless web service, a distributed database software is often sensitive to the instance auto-scaling. You need to make sure that there is a certain interval between each auto-scaling in case of too frequent scaling operations.
```